net-misc/teamviewer/teamviewer-15.20.6.ebuild

```
@@ -0,0 +1,166 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_MAJOR="$(ver_cut 1)"
+MY_P="${PN}${MY_MAJOR}"
+inherit desktop optfeature systemd xdg
```

Not /necessary/ but I prefer to avoid pre-inherit pollution when possible.
```suggestion
inherit desktop optfeature systemd xdg

MY_MAJOR="$(ver_cut 1)"
MY_P="${PN}${MY_MAJOR}"
```

---

I see, did not consider about this. 
Fixed.